docs/index.rst

```diff
@@ -44,18 +53,80 @@ Welcome to NEURON's documentation!
 .. toctree::
    :maxdepth: 1
    :caption: Removed Features
+   :hidden:
 
    removed_features.rst
 
 .. toctree::
    :maxdepth: 1
    :caption: Changelog
+   :hidden:
 
    changelog.md
 
-Indices and tables
-==================
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+.. image:: neuron-overview.png
+   :width: 75%
+   :align: center
+   :alt: NEURON image with ShapePlot, code, and cortical column
+
+
+Installation
+------------
+
+.. raw:: html
+
+   <div id="downloadMacOS" style="display:none">
+      <p>
+         On macOS, install via:
+         <code>pip3 install neuron</code>
+      </p>
+      <p>
+         Alternatively, 
+         <a href="https://github.com/neuronsimulator/nrn/releases/download/8.1.0/nrn-8.1.0-macosx-10.9-universal2-py-38-39-310.pkg">download macOS installer</a>
+      </p>
+   </div>
+   <div id="downloadWindows" style="display:none">
+      <a class="button" href="https://github.com/neuronsimulator/nrn/releases/download/8.1.0/nrn-8.1.0.w64-mingw-py-36-37-38-39-310-setup.exe">Download Windows installer (64 bit)</a>
```

Comment:
Would be nice to have a generic link that points to latest exe (thus avoiding updating the docs after each release)